src/Model/Table/QueuedJobsTable.php

```diff
@@ -190,15 +204,25 @@ public function getStats() {
 	public function requestJob(array $capabilities, $group = null) {
 		$now = new Time();
 		$nowStr = $now->toDateTimeString();
+		$driverName = $this->_getDriverName();
 
 		$query = $this->find();
+		$age = $query->newExpr()->add('IFNULL(TIMESTAMPDIFF(SECOND, "' . $nowStr . '", notbefore), 0)');
+		switch ($driverName) {
+			case 'Postgres':
+			case 'Mysql':
```

Comment:
Should we use class const here to avoid spelling mistakes?

Reply:
Either you put $age down here, or you complete remove those case statements as they are currently no-ops.

Reply:
Yes, I'll change tomorrow